graphql/typeDefs.ts

```diff
@@ -57,6 +57,12 @@ export default gql`
     ): Comment
     deleteComment(id: Int!): Comment
     addModule(lessonId: Int!, name: String!, content: String!): Module!
+    updateModule(
+      id: Int!
+      lessonId: Int!
+      name: String!
```

Comment:
I think `title` instead of `name` is more suitable.

Reply:
I want to keep it consistent with addModule.